hasura/metadata/databases/default/tables/public_claims.yaml

```diff
@@ -51,16 +51,20 @@ select_permissions:
     - txHash
     - updated_at
     filter:
-      distribution:
-        epoch:
-          circle:
-            users:
-              _and:
-              - deleted_at:
-                  _is_null: true
-              - profile:
-                  id:
-                    _eq: X-Hasura-User-Id
+      _or:
+      - profile:
+          id:
+            _eq: X-Hasura-User-Id
+      - distribution:
```

Comment:
Is this the correct reading of this permission?
`if the claim's profile's id eq X-Hasura-User-Id OR the claim's distribution's epoch's circle's users set contains a user row with both deleted_at null and profile id eq X-Hasura-User-Id`? I.e. I should be able to select (view) all claims for which I'm a member of the circle?

Reply:
Yeah, this is the incumbent perm and is needed to ensure the distribution tables and summaries render the appropriate data. We can lock this down in a separate issue if needed, but this is essentially public anyway, since it's onchain.